blocks/related-articles/related-articles.js

```diff
@@ -14,6 +15,20 @@ function getFilter(pageTags) {
   };
 }
 
+function getPlaceHolderValue(key, placeholders) {
+  const value = placeholders[toCamelCase(key)];
+  return value || '';
```

Comment:
This could be in one line and maybe the `getPlaceHolderValue` is not needed any more? Or should we move  `getPlaceHolderValue` to utils.js?

Reply:
Makes sense. Replaced function with inline expression.